assets/ExpressionMetadata.json

```diff
@@ -329,6 +329,12 @@
             "minimumArguments": 2,
             "maximumArguments": 2
         },
+        {
+            "name": "list",
+            "expectedUsage": "list(resourceName\/resourceIdentifier, apiVersion)",
+            "description": "Performs a GET call to a REST API on the given resource. The returned data depends on the resource type. See https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-resource#valid-uses.",
```

Comment:
Should we say?

... The function arguments and the returned data depend on the resource type...

Technically true for all of them, but this is likely to be the most common/ambiguous.